benchmark/scenarios/__init__.py

```diff
@@ -1,9 +1,15 @@
 from .no_load import NoLoadScenario
-from .read_request_load import (
-    ReadRequestLoadScenario, RequestRateTooLow, RequestRateNotReached,
-    RequestOverload
-)
+from .read_request_load import read_request_load_scenario
+
 from .write_request_load import (
-    WriteRequestLoadScenario, WRequestRateTooLow, WRequestRateNotReached,
-    WRequestOverload, DatasetCreationTimeout
+    write_request_load_scenario, DatasetCreationTimeout
+)
+from ._rate_measurer import (
+    RateMeasurer, DEFAULT_SAMPLE_SIZE
+)
+from ._request_load import (
+    RequestRateTooLow, RequestRateNotReached,
+    RequestOverload, NoNodesFound, RequestLoadScenario
 )
+
+
```

Comment:
Pyflakes has a lot to say about this module.  The general form of the error is pre-existing but you may as well fix it in this branch, I think.

The idiomatic way to declare that these names are all being imported for re-export is to put them into an `__all__` definition.  See `flocker/common/__init__.py` for an example.

Also this file is missing a module(/package) docstring and copyright header.